documentation/src/main/asciidoc/user_guide/persistence.adoc

```diff
@@ -241,6 +243,34 @@ This strategy is suited for cache stores with low latency and cheap operation co
 </persistence>
 ----
 
+==== Segmented Stores
+
+A store can optionally be segmented. That is that data is stored by the segment
+that its key maps to. Information pertaining to segments and the ownership of
+them can be found at <<key_ownership,Key Ownership>>.
+
+Segmenting a store has the benefit of increasing the read performance when all
+of the data that maps is used. This would be used for any stream operation
+performed upon the cache, size method as well as preloading and state
+transfer in a cache.
```

Comment:
I think the first sentence of this paragraph needs clarifying. I'm not sure what you're trying to say with "all of the data that maps is used".